### PR TITLE
AdpptOpenJDK 2020-04-14 Update

### PR DESCRIPTION
**Pull request by poad**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala [hunk 1]
```diff
@@ -345,4 +347,178 @@ class AdoptOpenJdkMigrations {
...
+      removeVersion("java", "8.0.242.j9-adpt", _)
+    )
+  }
+
+  @ChangeSet(
+    order = "0015",
+    id = "0015-add_adoptopenjdk-j9_14_0_1",
+    author = "poad"
+  )
+  def migrate0015(implicit db: MongoDatabase) = {
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala:495]: Is there a `hs` variant available for `14.0.1`?

Comment (poad) [reply src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala:495]: @marc0der 
No hotspot has been released for 14.0.1 yet.

Oops!
According to AdoptOpenJDK's Slack, it looks like it's going to be released today.

Comment (poad) [reply src/main/scala/io/sdkman/changelogs/java/AdoptOpenJdkMigrations.scala:495]: @marc0der 
14.0.1 of hotspot has been released, so I added it.
commit is https://github.com/sdkman/sdkman-db-migrations/pull/249/commits/1c08c64a63195c532650fba447822bafedda1453.

**Comment by marc0der** [comment]
Thanks @poad!

**Comment by poad** [comment]
Thanks @marc0der!

**Pull request merged**